src/compressed_chunk.c

```diff
@@ -216,16 +216,13 @@ size_t Compressed_DelRange(Chunk_t *chunk, timestamp_t startTs, timestamp_t endT
     return deleted_count;
 }
 
-// decompress chunk
-static Chunk *decompressChunk(const CompressedChunk *compressedChunk) {
-    assert(compressedChunk != NULL);
-
+static inline void decompressChunkInplace(const CompressedChunk *compressedChunk,
```

Comment:
@OfirMos inlining of large functions might be counterproductive. I suggest that the inlining change to be done on a different PR ( with a perf-* branch name so we can precisely validate that indeed this helps/hurts performance)

Reply:
@filipecosta90 I agree but the logic of this function was in line in the caller function of decompressChunkInplace, your remark is relevant to the caller function which is now inline: decompressChunk()

Reply:
@filipecosta90 decompressChunk() is being called only from one other function so I think that inlining it can't be counterproductive, the amount of space the code will take will stay the same.